static/app/views/settings/projectSourceMaps/detail/index.spec.tsx

```diff
@@ -18,6 +18,9 @@ describe('ProjectSourceMapsDetail', () => {
     params: {projectId: project.slug, name: archiveName},
```

Comment:
its possible to pass params to initializeOrg and get the routerProps back out all together

Reply:
Egh, I think I prefer just doing it from the router. Feels like initializeOrg is doing too many things already